lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -39,7 +39,6 @@ if test "x$suse_cloud_version" != "x5"; then
   die "Current installed version of SUSE Cloud is not 5: $suse_cloud_version"
 fi
 
-
```

Comment:
Tiny detail, but I think there are two empty lines before each "section" of the code (ie, bits starting with a `###` comment). Could you preserve this?